wiki/markdown/docker.md

```diff
@@ -18,29 +18,26 @@ With the Docker image user can try-out the Eiffel-Intelligence on a Docker Host
 
 ## Follow these step to build the Docker image.
 
-1. Build the Eiffel-intelligence war file:
-`mvn package -DskipTests`
+**1** Build the Eiffel-intelligence war file:
+    
+    mvn package -DskipTests
 
 
 This will produce a war file in the "target" folder.
 
-
-2. Build the Docker image with the war file that was produced from previous step:
-
-
-`docker build -t eiffel-intelligence-backend:1.0.1 --build-arg URL=./target/eiffel-intelligence-1.0.1.war -f src/main/docker/Dockerfile .`
-
+**2** Build the Docker image with the war file that was produced from previous step:
+    
+    docker build -t eiffel-intelligence-backend:1.0.1 --build-arg URL=./target/eiffel-intelligence-1.0.1.war -f src/main/docker/Dockerfile .
 
 Now docker image has build with tag "eiffel-intelligence-backend:1.0.1"
 
 ## Run Docker image on local Docker Host
 To run the produced docker image on the local Docker host, execute this command:
-
-
-`docker run -p 8070:8080 --expose 8080 -e server.port=8080 -e logging.level.log.level.root=DEBUG -e logging.level.org.springframework.web=DEBUG -e logging.level.com.ericsson.ei=DEBUG -e spring.data.mongodb.host=mongodb -e spring.data.mongodb.port=27017 eiffel-intelligence:1.0.1`
+    
+    docker run -p 8070:8080 --expose 8080 -e server.port=8080 -e logging.level.log.level.root=DEBUG -e logging.level.org.springframework.web=DEBUG -e logging.level.com.ericsson.ei=DEBUG -e spring.data.mongodb.host=mongodb -e spring.data.mongodb.port=27017 eiffel-intelligence:1.0.1
 
 MongoDB, RabbitMq and other Eiffel-Intelligence required components need to running and configured via these application properties that is provided to the docker command above. See the application.properties file for all available/required properties:
-[application.properties](https://github.com/Ericsson/eiffel-intelligence/blob/master/src/main/resources/application.properties)
+[application.properties](https://github.com/eiffel-community/eiffel-intelligence/blob/master/src/main/resources/application.properties)
 
 
 # Some info of all flags to this command
```

Comment:
How come this heading is larger than the previous ones? I feel it should be on the same level with '##' and the subheadings for it (Docker flags and Eiffel Intelligence Spring properties) should be '###'

Reply:
This header is headline for the two coming sub-headers which is about java properties flags and Docker flags. And "of" should be "about", will update.